test/unit/test_Sorts.cc

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2021, Antti Hyvarinen <antti.hyvarinen@gmail.com>
+ * Copyright (c) 2021, Martin Blicha <martin.blicha@gmail.com>
```

Comment:
It still feels funny to put my name in the file which I did not write.

Reply:
OK, I'll remove it here.